src/Microsoft.Management.Infrastructure.CimCmdlets/project.json

```diff
@@ -7,6 +7,7 @@
         "delaySign": true,
         "publicSign": false,
         "warningsAsErrors": true,
+        "debugType": "portable"
```

Comment:
This might be the wrong place to put this entry because this `buildOptions` section is shared by both `netstandard1.6` and `net451` target frameworks. This setting probably should be put in the `buildOptions` section within `netstandard1.6` section below.